cpp/src/gandiva/gdv_function_stubs.cc

```diff
@@ -183,6 +184,18 @@ bool gdv_fn_in_expr_lookup_utf8(int64_t ptr, const char* data, int data_len,
   return holder->HasValue(arrow::util::string_view(data, data_len));
 }
 
+int64_t gdv_fn_convert_timezone(int64_t ptr, const char* src_tz, int src_tz_len,
+                                const char* dst_tz, int dst_tz_len, int64_t src_millis,
+                                bool validity) {
```

Comment:
Why you add that last validity parameter?

Reply:
I uncommented the lines below